Time taken to run the queries on ./repositories/Django/thread14/ACMAS_ACMAS-Frontend/ACMAS-ACMAS-Frontend-d05a94a: 2049.096 seconds.
Analysis failed for: ./repositories/Django/thread14/AI4Bharat_Chitralekha-Backend/AI4Bharat-Chitralekha-Backend-9adfdca Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/AI4Bharat_Chitralekha-Backend/AI4Bharat-Chitralekha-Backend-9adfdca -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 2m38s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/AI4Bharat_Chitralekha-Backend/AI4Bharat-Chitralekha-Backend-9adfdca-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 25.6s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/AI4Bharat_Chitralekha-Backend/AI4Bharat-Chitralekha-Backend-9adfdca-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/AI4Bharat_Chitralekha-Backend/AI4Bharat-Chitralekha-Backend-9adfdca: 346.072 seconds.
Analysis failed for: ./repositories/Django/thread14/AcceleratorYandexPracticum_family_budget/AcceleratorYandexPracticum-family_budget-b2fd589 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/AcceleratorYandexPracticum_family_budget/AcceleratorYandexPracticum-family_budget-b2fd589 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m10s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/AcceleratorYandexPracticum_family_budget/AcceleratorYandexPracticum-family_budget-b2fd589-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 11.1s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/AcceleratorYandexPracticum_family_budget/AcceleratorYandexPracticum-family_budget-b2fd589-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/AcceleratorYandexPracticum_family_budget/AcceleratorYandexPracticum-family_budget-b2fd589: 185.013 seconds.
Analysis failed for: ./repositories/Django/thread14/Artsdatabanken_okologiskegrunnkart/Artsdatabanken-okologiskegrunnkart-0826815 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/Artsdatabanken_okologiskegrunnkart/Artsdatabanken-okologiskegrunnkart-0826815 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m27s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/Artsdatabanken_okologiskegrunnkart/Artsdatabanken-okologiskegrunnkart-0826815-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 14.9s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/Artsdatabanken_okologiskegrunnkart/Artsdatabanken-okologiskegrunnkart-0826815-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/Artsdatabanken_okologiskegrunnkart/Artsdatabanken-okologiskegrunnkart-0826815: 229.039 seconds.
Time taken to run the queries on ./repositories/Django/thread14/Arvind-4_E-Commerce/Arvind-4-E-Commerce-0bff5ff: 2003.004 seconds.
Analysis failed for: ./repositories/Django/thread14/AudioCommons_ac-mediator/AudioCommons-ac-mediator-3201c12 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/AudioCommons_ac-mediator/AudioCommons-ac-mediator-3201c12 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m19s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/AudioCommons_ac-mediator/AudioCommons-ac-mediator-3201c12-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 9.2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/AudioCommons_ac-mediator/AudioCommons-ac-mediator-3201c12-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/AudioCommons_ac-mediator/AudioCommons-ac-mediator-3201c12: 220.693 seconds.
Analysis failed for: ./repositories/Django/thread14/City-of-Helsinki_parkkihubi/City-of-Helsinki-parkkihubi-d8a86c5 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/City-of-Helsinki_parkkihubi/City-of-Helsinki-parkkihubi-d8a86c5 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m24s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/City-of-Helsinki_parkkihubi/City-of-Helsinki-parkkihubi-d8a86c5-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 17s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/City-of-Helsinki_parkkihubi/City-of-Helsinki-parkkihubi-d8a86c5-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/City-of-Helsinki_parkkihubi/City-of-Helsinki-parkkihubi-d8a86c5: 244.283 seconds.
Analysis failed for: ./repositories/Django/thread14/Cranio-worldwide_backend/Cranio-worldwide-backend-be0c723 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/Cranio-worldwide_backend/Cranio-worldwide-backend-be0c723 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m19s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/Cranio-worldwide_backend/Cranio-worldwide-backend-be0c723-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 11.9s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/Cranio-worldwide_backend/Cranio-worldwide-backend-be0c723-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/Cranio-worldwide_backend/Cranio-worldwide-backend-be0c723: 210.359 seconds.
Analysis failed for: ./repositories/Django/thread14/Debakel_Dumpstermap/Debakel-Dumpstermap-ca17a81 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/Debakel_Dumpstermap/Debakel-Dumpstermap-ca17a81 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m17s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/Debakel_Dumpstermap/Debakel-Dumpstermap-ca17a81-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 9.6s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/Debakel_Dumpstermap/Debakel-Dumpstermap-ca17a81-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/Debakel_Dumpstermap/Debakel-Dumpstermap-ca17a81: 213.311 seconds.
Time taken to run the queries on ./repositories/Django/thread14/Enterprise-CMCS_cmcs-eregulations/Enterprise-CMCS-cmcs-eregulations-df64b54: 2416.182 seconds.
Time taken to run the queries on ./repositories/Django/thread14/EscrutinioSocial_escrutinio-social/EscrutinioSocial-escrutinio-social-ec2d2fc: 2435.504 seconds.
Time taken to run the queries on ./repositories/Django/thread14/FuAdmin_fu-admin/FuAdmin-fu-admin-9904ab8: 2258.93 seconds.
Time taken to run the queries on ./repositories/Django/thread14/GrafeasGroup_blossom/GrafeasGroup-blossom-4103723: 2418.368 seconds.
Time taken to run the queries on ./repositories/Django/thread14/GreatDevelopers_LibreHatti/GreatDevelopers-LibreHatti-bb0e02c: 2001.979 seconds.
Analysis failed for: ./repositories/Django/thread14/Hirochon_Shappar/Hirochon-Shappar-053b200 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/Hirochon_Shappar/Hirochon-Shappar-053b200 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m4s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/Hirochon_Shappar/Hirochon-Shappar-053b200-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 12.3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/Hirochon_Shappar/Hirochon-Shappar-053b200-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/Hirochon_Shappar/Hirochon-Shappar-053b200: 182.268 seconds.
Analysis failed for: ./repositories/Django/thread14/HyunSangHan_MeetingTime/HyunSangHan-MeetingTime-84a99b3 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/HyunSangHan_MeetingTime/HyunSangHan-MeetingTime-84a99b3 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m17s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/HyunSangHan_MeetingTime/HyunSangHan-MeetingTime-84a99b3-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 11.4s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/HyunSangHan_MeetingTime/HyunSangHan-MeetingTime-84a99b3-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/HyunSangHan_MeetingTime/HyunSangHan-MeetingTime-84a99b3: 216.09 seconds.
Analysis failed for: ./repositories/Django/thread14/Join2Gather_WeMeet/Join2Gather-WeMeet-a0e6086 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/Join2Gather_WeMeet/Join2Gather-WeMeet-a0e6086 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 2m16s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/Join2Gather_WeMeet/Join2Gather-WeMeet-a0e6086-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 17s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/Join2Gather_WeMeet/Join2Gather-WeMeet-a0e6086-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/Join2Gather_WeMeet/Join2Gather-WeMeet-a0e6086: 311.073 seconds.
Analysis failed for: ./repositories/Django/thread14/Kaniwani_kw-backend/Kaniwani-kw-backend-58faf19 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/Kaniwani_kw-backend/Kaniwani-kw-backend-58faf19 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m39s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/Kaniwani_kw-backend/Kaniwani-kw-backend-58faf19-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 13.1s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/Kaniwani_kw-backend/Kaniwani-kw-backend-58faf19-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/Kaniwani_kw-backend/Kaniwani-kw-backend-58faf19: 231.203 seconds.
Time taken to run the queries on ./repositories/Django/thread14/LoRexxar_Kunlun-M/LoRexxar-Kunlun-M-fbd30dc: 2443.97 seconds.
Analysis failed for: ./repositories/Django/thread14/Longwelwind_swords-and-ravens/Longwelwind-swords-and-ravens-9819475 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/Longwelwind_swords-and-ravens/Longwelwind-swords-and-ravens-9819475 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m12s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/Longwelwind_swords-and-ravens/Longwelwind-swords-and-ravens-9819475-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 11.3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/Longwelwind_swords-and-ravens/Longwelwind-swords-and-ravens-9819475-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/Longwelwind_swords-and-ravens/Longwelwind-swords-and-ravens-9819475: 203.404 seconds.
Analysis failed for: ./repositories/Django/thread14/MPMG-DCC-UFMG_C01/MPMG-DCC-UFMG-C01-cac2014 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/MPMG-DCC-UFMG_C01/MPMG-DCC-UFMG-C01-cac2014 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 2m31s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/MPMG-DCC-UFMG_C01/MPMG-DCC-UFMG-C01-cac2014-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 16.2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/MPMG-DCC-UFMG_C01/MPMG-DCC-UFMG-C01-cac2014-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/MPMG-DCC-UFMG_C01/MPMG-DCC-UFMG-C01-cac2014: 334.771 seconds.
Time taken to run the queries on ./repositories/Django/thread14/Mercado-Social-de-Madrid_gestionMES/Mercado-Social-de-Madrid-gestionMES-e751576: 2366.706 seconds.
Analysis failed for: ./repositories/Django/thread14/NaNShaner_repoll/NaNShaner-repoll-3f754e3 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/NaNShaner_repoll/NaNShaner-repoll-3f754e3 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m18s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/NaNShaner_repoll/NaNShaner-repoll-3f754e3-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 10.3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/NaNShaner_repoll/NaNShaner-repoll-3f754e3-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/NaNShaner_repoll/NaNShaner-repoll-3f754e3: 207.267 seconds.
Time taken to run the queries on ./repositories/Django/thread14/OnToology_OnToology/OnToology-OnToology-b63a296: 2329.566 seconds.
Time taken to run the queries on ./repositories/Django/thread14/OpenBankProject_API-Manager/OpenBankProject-API-Manager-030e50c: 2317.629 seconds.
Analysis failed for: ./repositories/Django/thread14/Parisson_Telemeta/Parisson-Telemeta-c47d9c1 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/Parisson_Telemeta/Parisson-Telemeta-c47d9c1 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 2m12s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/Parisson_Telemeta/Parisson-Telemeta-c47d9c1-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 21.3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/Parisson_Telemeta/Parisson-Telemeta-c47d9c1-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/Parisson_Telemeta/Parisson-Telemeta-c47d9c1: 313.929 seconds.
Analysis failed for: ./repositories/Django/thread14/PyBAQ_temii/PyBAQ-temii-b6478cf Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/PyBAQ_temii/PyBAQ-temii-b6478cf -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 52.3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/PyBAQ_temii/PyBAQ-temii-b6478cf-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 9.8s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/PyBAQ_temii/PyBAQ-temii-b6478cf-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/PyBAQ_temii/PyBAQ-temii-b6478cf: 168.864 seconds.
Time taken to run the queries on ./repositories/Django/thread14/RadicalxChange_rxc-voice/RadicalxChange-rxc-voice-e7936cf: 2287.95 seconds.
Time taken to run the queries on ./repositories/Django/thread14/ScanOC_trunk-player/ScanOC-trunk-player-3eba914: 2362.983 seconds.
Analysis failed for: ./repositories/Django/thread14/SeonminKim1_SidePro-BE/SeonminKim1-PJ-SidePro_BE-1b6ff67 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/SeonminKim1_SidePro-BE/SeonminKim1-PJ-SidePro_BE-1b6ff67 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 59.6s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/SeonminKim1_SidePro-BE/SeonminKim1-PJ-SidePro_BE-1b6ff67-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 14.3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/SeonminKim1_SidePro-BE/SeonminKim1-PJ-SidePro_BE-1b6ff67-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/SeonminKim1_SidePro-BE/SeonminKim1-PJ-SidePro_BE-1b6ff67: 183.835 seconds.
Time taken to run the queries on ./repositories/Django/thread14/SuperCoopBerlin_tapir/SuperCoopBerlin-tapir-6036c3b: 2106.387 seconds.
Time taken to run the queries on ./repositories/Django/thread14/TencentBlueKing_bk-goods-application/TencentBlueKing-bk-goods-application-8744f85: 2316.78 seconds.
Time taken to run the queries on ./repositories/Django/thread14/TencentBlueKing_bk-iam-saas/TencentBlueKing-bk-iam-saas-3fc702d: 2509.139 seconds.
Analysis failed for: ./repositories/Django/thread14/TencentBlueKing_bk-monitor/TencentBlueKing-bk-monitor-841c52d Reason: Error: spawnSync /bin/sh ENOBUFS
Could not delete database for: ./repositories/Django/thread14/TencentBlueKing_bk-monitor/TencentBlueKing-bk-monitor-841c52d Reason: Error: ENOTEMPTY: directory not empty, rmdir './repositories/Django/thread14/TencentBlueKing_bk-monitor/TencentBlueKing-bk-monitor-841c52d-database/src/home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/TencentBlueKing_bk-monitor/TencentBlueKing-bk-monitor-841c52d/bklog/iam/contrib'
Time taken to run the queries on ./repositories/Django/thread14/TencentBlueKing_bk-monitor/TencentBlueKing-bk-monitor-841c52d: 111.703 seconds.
Time taken to run the queries on ./repositories/Django/thread14/Tortoise-Community_Backend/Tortoise-Community-Backend-38e581f: 1989.054 seconds.
Time taken to run the queries on ./repositories/Django/thread14/UUDigitalHumanitieslab_timealign/UUDigitalHumanitieslab-timealign-8aebf6e: 2646.976 seconds.
Analysis failed for: ./repositories/Django/thread14/Web-Multi-Media_HttpStreamingServer/Web-Multi-Media-HttpStreamingServer-703c831 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/Web-Multi-Media_HttpStreamingServer/Web-Multi-Media-HttpStreamingServer-703c831 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m17s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/Web-Multi-Media_HttpStreamingServer/Web-Multi-Media-HttpStreamingServer-703c831-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 15.5s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/Web-Multi-Media_HttpStreamingServer/Web-Multi-Media-HttpStreamingServer-703c831-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/Web-Multi-Media_HttpStreamingServer/Web-Multi-Media-HttpStreamingServer-703c831: 222.524 seconds.
Analysis failed for: ./repositories/Django/thread14/WikipediaLibrary_externallinks/WikipediaLibrary-externallinks-01a0b0d Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/WikipediaLibrary_externallinks/WikipediaLibrary-externallinks-01a0b0d -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m30s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/WikipediaLibrary_externallinks/WikipediaLibrary-externallinks-01a0b0d-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 10.8s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/WikipediaLibrary_externallinks/WikipediaLibrary-externallinks-01a0b0d-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/WikipediaLibrary_externallinks/WikipediaLibrary-externallinks-01a0b0d: 238.01 seconds.
Analysis failed for: ./repositories/Django/thread14/WuraLab_DNA_backend/WuraLab-DNA_backend-1af653e Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/WuraLab_DNA_backend/WuraLab-DNA_backend-1af653e -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m19s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/WuraLab_DNA_backend/WuraLab-DNA_backend-1af653e-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 12.6s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/WuraLab_DNA_backend/WuraLab-DNA_backend-1af653e-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/WuraLab_DNA_backend/WuraLab-DNA_backend-1af653e: 208.585 seconds.
Time taken to run the queries on ./repositories/Django/thread14/a1401358759_my_site/a1401358759-my_site-001517f: 2360.861 seconds.
Analysis failed for: ./repositories/Django/thread14/abhisheksaxena1998_Malicious-Urlv5/abhisheksaxena1998-Malicious-Urlv5-29631c7 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/abhisheksaxena1998_Malicious-Urlv5/abhisheksaxena1998-Malicious-Urlv5-29631c7 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m28s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/abhisheksaxena1998_Malicious-Urlv5/abhisheksaxena1998-Malicious-Urlv5-29631c7-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 9s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/abhisheksaxena1998_Malicious-Urlv5/abhisheksaxena1998-Malicious-Urlv5-29631c7-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/abhisheksaxena1998_Malicious-Urlv5/abhisheksaxena1998-Malicious-Urlv5-29631c7: 231.788 seconds.
Analysis failed for: ./repositories/Django/thread14/alexzawadsky_Grocket/alexzawadsky-Grocket-e4863a8 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/alexzawadsky_Grocket/alexzawadsky-Grocket-e4863a8 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m36s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/alexzawadsky_Grocket/alexzawadsky-Grocket-e4863a8-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 15.2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/alexzawadsky_Grocket/alexzawadsky-Grocket-e4863a8-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/alexzawadsky_Grocket/alexzawadsky-Grocket-e4863a8: 230.293 seconds.
Time taken to run the queries on ./repositories/Django/thread14/aliss_ALISS/aliss-ALISS-5e77b26: 2390.499 seconds.
Time taken to run the queries on ./repositories/Django/thread14/artoonie_rcvis/artoonie-rcvis-0cf4399: 2384.665 seconds.
Time taken to run the queries on ./repositories/Django/thread14/avantifellows_plio-backend/avantifellows-plio-backend-530721f: 2031.827 seconds.
Analysis failed for: ./repositories/Django/thread14/aybruhm_authentication-service-be/aybruhm-authentication-service-be-0135773 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/aybruhm_authentication-service-be/aybruhm-authentication-service-be-0135773 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m17s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/aybruhm_authentication-service-be/aybruhm-authentication-service-be-0135773-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 12.7s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/aybruhm_authentication-service-be/aybruhm-authentication-service-be-0135773-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/aybruhm_authentication-service-be/aybruhm-authentication-service-be-0135773: 197.517 seconds.
Analysis failed for: ./repositories/Django/thread14/bartromgens_care/bartromgens-care-cc69568 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/bartromgens_care/bartromgens-care-cc69568 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m1s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/bartromgens_care/bartromgens-care-cc69568-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 12.4s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/bartromgens_care/bartromgens-care-cc69568-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/bartromgens_care/bartromgens-care-cc69568: 180.302 seconds.
Time taken to run the queries on ./repositories/Django/thread14/baseplate-admin_CoreProject/coreproject-moe-CoreProject-Monorepo-ed35475: 2554.305 seconds.
Analysis failed for: ./repositories/Django/thread14/betagouv_CNR/betagouv-CNR-9c4364e Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/betagouv_CNR/betagouv-CNR-9c4364e -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m14s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/betagouv_CNR/betagouv-CNR-9c4364e-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 7s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/betagouv_CNR/betagouv-CNR-9c4364e-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/betagouv_CNR/betagouv-CNR-9c4364e: 186.012 seconds.
Time taken to run the queries on ./repositories/Django/thread14/bihealth_sodar-server/bihealth-sodar-server-cad197d: 2705.455 seconds.
Time taken to run the queries on ./repositories/Django/thread14/c-base_c-beam/c-base-c-beam-74e2e21: 2241.162 seconds.
Time taken to run the queries on ./repositories/Django/thread14/camerahub_camerahub/camerahub-camerahub-5889383: 2572.92 seconds.
Time taken to run the queries on ./repositories/Django/thread14/chateval_application/chateval-application-cbcf49c: 2943.277 seconds.
Analysis failed for: ./repositories/Django/thread14/cityofaustin_joplin/cityofaustin-joplin-01424e4 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/cityofaustin_joplin/cityofaustin-joplin-01424e4 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m35s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/cityofaustin_joplin/cityofaustin-joplin-01424e4-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 15.1s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/cityofaustin_joplin/cityofaustin-joplin-01424e4-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/cityofaustin_joplin/cityofaustin-joplin-01424e4: 239.782 seconds.
Analysis failed for: ./repositories/Django/thread14/codefordenver_waterbeacon/codefordenver-waterbeacon-9349458 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/codefordenver_waterbeacon/codefordenver-waterbeacon-9349458 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m29s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/codefordenver_waterbeacon/codefordenver-waterbeacon-9349458-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 11.1s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/codefordenver_waterbeacon/codefordenver-waterbeacon-9349458-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/codefordenver_waterbeacon/codefordenver-waterbeacon-9349458: 234.98 seconds.
Time taken to run the queries on ./repositories/Django/thread14/codervivek5_VigyBag/codervivek5-VigyBag-4910697: 2009.512 seconds.
Time taken to run the queries on ./repositories/Django/thread14/compute-tooling_compute-studio/compute-tooling-compute-studio-0e9b53a: 2591.638 seconds.
Time taken to run the queries on ./repositories/Django/thread14/coronasafe_care/coronasafe-care-12c645d: 2640.535 seconds.
Analysis failed for: ./repositories/Django/thread14/crawler-studio_crawler-studio/crawler-studio-crawler-studio-ecfc371 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/crawler-studio_crawler-studio/crawler-studio-crawler-studio-ecfc371 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m13s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/crawler-studio_crawler-studio/crawler-studio-crawler-studio-ecfc371-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 16.6s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/crawler-studio_crawler-studio/crawler-studio-crawler-studio-ecfc371-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/crawler-studio_crawler-studio/crawler-studio-crawler-studio-ecfc371: 210.026 seconds.
Time taken to run the queries on ./repositories/Django/thread14/cyllab_CalcUS/cyllab-CalcUS-12d6acd: 2747.156 seconds.
Time taken to run the queries on ./repositories/Django/thread14/datahaven-net_zenaida/datahaven-net-zenaida-f75df14: 2464.923 seconds.
Analysis failed for: ./repositories/Django/thread14/devlup-labs_woc/devlup-labs-woc-d01e94a Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/devlup-labs_woc/devlup-labs-woc-d01e94a -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/devlup-labs_woc/devlup-labs-woc-d01e94a-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 10.6s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/devlup-labs_woc/devlup-labs-woc-d01e94a-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/devlup-labs_woc/devlup-labs-woc-d01e94a: 196.862 seconds.
Time taken to run the queries on ./repositories/Django/thread14/djaodjin_djaodjin-extended-templates/djaodjin-djaodjin-extended-templates-ba8b10e: 2312.981 seconds.
Analysis failed for: ./repositories/Django/thread14/doableware_djongo/doableware-djongo-421cba3 Reason: Error: spawnSync /bin/sh ENOBUFS
Could not delete database for: ./repositories/Django/thread14/doableware_djongo/doableware-djongo-421cba3 Reason: Error: ENOTEMPTY: directory not empty, rmdir './repositories/Django/thread14/doableware_djongo/doableware-djongo-421cba3-database/src/home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/doableware_djongo/doableware-djongo-421cba3/tests/django_tests/tests/v21/tests'
Time taken to run the queries on ./repositories/Django/thread14/doableware_djongo/doableware-djongo-421cba3: 113.269 seconds.
Analysis failed for: ./repositories/Django/thread14/duo-labs_webauthn.io/duo-labs-webauthn.io-695d865 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/duo-labs_webauthn.io/duo-labs-webauthn.io-695d865 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m4s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/duo-labs_webauthn.io/duo-labs-webauthn.io-695d865-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 8.4s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/duo-labs_webauthn.io/duo-labs-webauthn.io-695d865-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/duo-labs_webauthn.io/duo-labs-webauthn.io-695d865: 188.592 seconds.
Analysis failed for: ./repositories/Django/thread14/encukou_freezeyt/encukou-freezeyt-28f2f7e Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/encukou_freezeyt/encukou-freezeyt-28f2f7e -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m16s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/encukou_freezeyt/encukou-freezeyt-28f2f7e-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 8s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/encukou_freezeyt/encukou-freezeyt-28f2f7e-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/encukou_freezeyt/encukou-freezeyt-28f2f7e: 208.366 seconds.
Time taken to run the queries on ./repositories/Django/thread14/erudit_eruditorg/erudit-eruditorg-d953bf9: 2627.249 seconds.
Analysis failed for: ./repositories/Django/thread14/fecgov_fec-cms/fecgov-fec-cms-853e301 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/fecgov_fec-cms/fecgov-fec-cms-853e301 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m26s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/fecgov_fec-cms/fecgov-fec-cms-853e301-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 14.4s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/fecgov_fec-cms/fecgov-fec-cms-853e301-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/fecgov_fec-cms/fecgov-fec-cms-853e301: 239.596 seconds.
Analysis failed for: ./repositories/Django/thread14/fedspendingtransparency_usaspending-api/fedspendingtransparency-usaspending-api-5304f88 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/fedspendingtransparency_usaspending-api/fedspendingtransparency-usaspending-api-5304f88 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 2m41s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/fedspendingtransparency_usaspending-api/fedspendingtransparency-usaspending-api-5304f88-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 32s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/fedspendingtransparency_usaspending-api/fedspendingtransparency-usaspending-api-5304f88-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/fedspendingtransparency_usaspending-api/fedspendingtransparency-usaspending-api-5304f88: 388.649 seconds.
Time taken to run the queries on ./repositories/Django/thread14/fra-uas-project_softDsim/fra-uas-project-softDsim-a6aa574: 2702.626 seconds.
Analysis failed for: ./repositories/Django/thread14/freedomofpress_pressfreedomtracker.us/freedomofpress-pressfreedomtracker.us-6a5c931 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/freedomofpress_pressfreedomtracker.us/freedomofpress-pressfreedomtracker.us-6a5c931 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m44s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/freedomofpress_pressfreedomtracker.us/freedomofpress-pressfreedomtracker.us-6a5c931-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 15s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/freedomofpress_pressfreedomtracker.us/freedomofpress-pressfreedomtracker.us-6a5c931-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/freedomofpress_pressfreedomtracker.us/freedomofpress-pressfreedomtracker.us-6a5c931: 263.529 seconds.
Analysis failed for: ./repositories/Django/thread14/freedompraise_ProjectSpark/freedompraise-ProjectSpark-259f7ef Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/freedompraise_ProjectSpark/freedompraise-ProjectSpark-259f7ef -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 59.7s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/freedompraise_ProjectSpark/freedompraise-ProjectSpark-259f7ef-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 7.1s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/freedompraise_ProjectSpark/freedompraise-ProjectSpark-259f7ef-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/freedompraise_ProjectSpark/freedompraise-ProjectSpark-259f7ef: 182.262 seconds.
Time taken to run the queries on ./repositories/Django/thread14/garg3133_JagratiWebApp/garg3133-JagratiWebApp-1ee8b6e: 2037.522 seconds.
Analysis failed for: ./repositories/Django/thread14/generator-meme_generator-meme/generator-meme-generator-meme-d24a146 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/generator-meme_generator-meme/generator-meme-generator-meme-d24a146 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m20s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/generator-meme_generator-meme/generator-meme-generator-meme-d24a146-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 12.4s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/generator-meme_generator-meme/generator-meme-generator-meme-d24a146-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/generator-meme_generator-meme/generator-meme-generator-meme-d24a146: 218.156 seconds.
Time taken to run the queries on ./repositories/Django/thread14/globocom_vault/globocom-vault-7e529bf: 2434.567 seconds.
Analysis failed for: ./repositories/Django/thread14/hpbl_WRITEME/hpbl-WRITEME-d1b401d Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/hpbl_WRITEME/hpbl-WRITEME-d1b401d -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 2m0s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/hpbl_WRITEME/hpbl-WRITEME-d1b401d-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 16.8s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/hpbl_WRITEME/hpbl-WRITEME-d1b401d-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/hpbl_WRITEME/hpbl-WRITEME-d1b401d: 284.359 seconds.
Analysis failed for: ./repositories/Django/thread14/iggy_simone/iggy-simone-eec1c1d Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/iggy_simone/iggy-simone-eec1c1d -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m7s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/iggy_simone/iggy-simone-eec1c1d-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 12.9s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/iggy_simone/iggy-simone-eec1c1d-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/iggy_simone/iggy-simone-eec1c1d: 194.784 seconds.
Analysis failed for: ./repositories/Django/thread14/jadolg_shadowmere/jadolg-shadowmere-863cf7d Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/jadolg_shadowmere/jadolg-shadowmere-863cf7d -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m26s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/jadolg_shadowmere/jadolg-shadowmere-863cf7d-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 11.7s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/jadolg_shadowmere/jadolg-shadowmere-863cf7d-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/jadolg_shadowmere/jadolg-shadowmere-863cf7d: 233.379 seconds.
Analysis failed for: ./repositories/Django/thread14/jazzband_sorl-thumbnail/jazzband-sorl-thumbnail-16dcfc8 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/jazzband_sorl-thumbnail/jazzband-sorl-thumbnail-16dcfc8 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 2m4s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/jazzband_sorl-thumbnail/jazzband-sorl-thumbnail-16dcfc8-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 11.9s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/jazzband_sorl-thumbnail/jazzband-sorl-thumbnail-16dcfc8-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/jazzband_sorl-thumbnail/jazzband-sorl-thumbnail-16dcfc8: 299.409 seconds.
Analysis failed for: ./repositories/Django/thread14/kookmin-sw_capstone-2020-6/kookmin-sw-capstone-2020-6-b1bd05f Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/kookmin-sw_capstone-2020-6/kookmin-sw-capstone-2020-6-b1bd05f -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 2m28s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/kookmin-sw_capstone-2020-6/kookmin-sw-capstone-2020-6-b1bd05f-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 18.2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/kookmin-sw_capstone-2020-6/kookmin-sw-capstone-2020-6-b1bd05f-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/kookmin-sw_capstone-2020-6/kookmin-sw-capstone-2020-6-b1bd05f: 313.817 seconds.
Time taken to run the queries on ./repositories/Django/thread14/ksiuwr_zosia16-site/ksiuwr-zosia16-site-e81882f: 2108.312 seconds.
Time taken to run the queries on ./repositories/Django/thread14/lukas2511_voctoconf/lukas2511-voctoconf-7c9d05e: 2272.354 seconds.
Analysis failed for: ./repositories/Django/thread14/maneike_codeStats/maneike-codeStats-c541553 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/maneike_codeStats/maneike-codeStats-c541553 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 2m0s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/maneike_codeStats/maneike-codeStats-c541553-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 11.2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/maneike_codeStats/maneike-codeStats-c541553-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/maneike_codeStats/maneike-codeStats-c541553: 284.775 seconds.
Time taken to run the queries on ./repositories/Django/thread14/matacoder_wildberries-rest-api/matacoder-wildberries-rest-api-e722fd6: 2281.132 seconds.
Analysis failed for: ./repositories/Django/thread14/mathislucka_dualtext/mathislucka-dualtext-0bbfc5f Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/mathislucka_dualtext/mathislucka-dualtext-0bbfc5f -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m35s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/mathislucka_dualtext/mathislucka-dualtext-0bbfc5f-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 15.3s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/mathislucka_dualtext/mathislucka-dualtext-0bbfc5f-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/mathislucka_dualtext/mathislucka-dualtext-0bbfc5f: 242.037 seconds.
Analysis failed for: ./repositories/Django/thread14/maykinmedia_objects-api/maykinmedia-objects-api-2080078 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/maykinmedia_objects-api/maykinmedia-objects-api-2080078 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m35s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/maykinmedia_objects-api/maykinmedia-objects-api-2080078-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 10.9s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/maykinmedia_objects-api/maykinmedia-objects-api-2080078-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/maykinmedia_objects-api/maykinmedia-objects-api-2080078: 242.815 seconds.
Time taken to run the queries on ./repositories/Django/thread14/mediacms-io_mediacms/mediacms-io-mediacms-c5047d8: 2439.059 seconds.
Analysis failed for: ./repositories/Django/thread14/metagov_gateway/metagov-gateway-9197896 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/metagov_gateway/metagov-gateway-9197896 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m23s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/metagov_gateway/metagov-gateway-9197896-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 15.2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/metagov_gateway/metagov-gateway-9197896-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/metagov_gateway/metagov-gateway-9197896: 235.678 seconds.
Analysis failed for: ./repositories/Django/thread14/nationalarchives_ds-caselaw-public-ui/nationalarchives-ds-caselaw-public-ui-ee07e43 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/nationalarchives_ds-caselaw-public-ui/nationalarchives-ds-caselaw-public-ui-ee07e43 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m42s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/nationalarchives_ds-caselaw-public-ui/nationalarchives-ds-caselaw-public-ui-ee07e43-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 14.1s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/nationalarchives_ds-caselaw-public-ui/nationalarchives-ds-caselaw-public-ui-ee07e43-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/nationalarchives_ds-caselaw-public-ui/nationalarchives-ds-caselaw-public-ui-ee07e43: 246.656 seconds.
Time taken to run the queries on ./repositories/Django/thread14/newtheatre_nt-tickets/newtheatre-nt-tickets-8b26b30: 1974.188 seconds.
Time taken to run the queries on ./repositories/Django/thread14/nezl0i_BlackPearl/nezl0i-BlackPearl-0b015f3: 2140.273 seconds.
Time taken to run the queries on ./repositories/Django/thread14/onosendi_social/onosendi-social-a2491c6: 2041.172 seconds.
Time taken to run the queries on ./repositories/Django/thread14/palfrey_wharf/palfrey-wharf-9024b2e: 2255.97 seconds.
Analysis failed for: ./repositories/Django/thread14/paucazou_theochrone/paucazou-theochrone-e5c3a06 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/paucazou_theochrone/paucazou-theochrone-e5c3a06 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m33s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/paucazou_theochrone/paucazou-theochrone-e5c3a06-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 12s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/paucazou_theochrone/paucazou-theochrone-e5c3a06-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/paucazou_theochrone/paucazou-theochrone-e5c3a06: 247.015 seconds.
Analysis failed for: ./repositories/Django/thread14/pinax_pinax-ratings/pinax-pinax-ratings-8f302ea Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/pinax_pinax-ratings/pinax-pinax-ratings-8f302ea -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m1s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/pinax_pinax-ratings/pinax-pinax-ratings-8f302ea-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 13.2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/pinax_pinax-ratings/pinax-pinax-ratings-8f302ea-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/pinax_pinax-ratings/pinax-pinax-ratings-8f302ea: 189.003 seconds.
Time taken to run the queries on ./repositories/Django/thread14/rackerlabs_scantron/rackerlabs-scantron-175f93c: 2339.576 seconds.
Time taken to run the queries on ./repositories/Django/thread14/ramonrdm_glossariolibras/ramonrdm-glossariolibras-8b1634e: 2006.921 seconds.
Time taken to run the queries on ./repositories/Django/thread14/ritstudentgovernment_PawPrints/ritstudentgovernment-PawPrints-0018222: 2056.632 seconds.
Analysis failed for: ./repositories/Django/thread14/s4dhulabs_vimana-framework/s4dhulabs-vimana-framework-d26d927 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/s4dhulabs_vimana-framework/s4dhulabs-vimana-framework-d26d927 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m44s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/s4dhulabs_vimana-framework/s4dhulabs-vimana-framework-d26d927-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 10.8s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/s4dhulabs_vimana-framework/s4dhulabs-vimana-framework-d26d927-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/s4dhulabs_vimana-framework/s4dhulabs-vimana-framework-d26d927: 262.763 seconds.
Time taken to run the queries on ./repositories/Django/thread14/shan18_Stock-Bridge/shan18-Stock-Bridge-ab54174: 2478.857 seconds.
Time taken to run the queries on ./repositories/Django/thread14/swarfarm_swarfarm/swarfarm-swarfarm-064e64d: 2308.905 seconds.
Analysis failed for: ./repositories/Django/thread14/teamhephy_controller/teamhephy-controller-a04cf88 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/teamhephy_controller/teamhephy-controller-a04cf88 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m40s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/teamhephy_controller/teamhephy-controller-a04cf88-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 17.2s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/teamhephy_controller/teamhephy-controller-a04cf88-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/teamhephy_controller/teamhephy-controller-a04cf88: 244.145 seconds.
Time taken to run the queries on ./repositories/Django/thread14/unicef_etools-partner-reporting-portal/unicef-etools-partner-reporting-portal-e717781: 2242.778 seconds.
Analysis failed for: ./repositories/Django/thread14/wagtail_wagtail-localize/wagtail-wagtail-localize-f005601 Reason: Error: Command failed: npm start -- -s=./repositories/Django/thread14/wagtail_wagtail-localize/wagtail-wagtail-localize-f005601 -l=python -t=11 -ct=14
Compiling query plan for /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Flask_Queries/Flask-login-is-used-check/flask_library_used_check.ql.
[1/1 eval 1m22s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/wagtail_wagtail-localize/wagtail-wagtail-localize-f005601-results/flask_library_used_check.bqrs.
Shutting down query evaluator.
Compiling query plan for /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1] Found in cache: /home/878352/Session-Management-Analyser/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
Starting evaluation of asterius27/codeql-extra-queries-python/Django_Queries/Django-auth-is-used-check/django_library_used_check.ql.
[1/1 eval 13.5s] Evaluation done; writing results to /home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/repositories/Django/thread14/wagtail_wagtail-localize/wagtail-wagtail-localize-f005601-results/django_library_used_check.bqrs.
Shutting down query evaluator.
file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97
        throw new Error("None of the supported libraries/frameworks is used");
              ^

Error: None of the supported libraries/frameworks is used
    at pythonAnalysis (file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/python-single-analysis.js:97:15)
    at file:///home/878352/Session-Management-Analyser/SessionManagementAnalyserApp/app.js:57:5
    at ModuleJob.run (node:internal/modules/esm/module_job:217:25)
    at async ModuleLoader.import (node:internal/modules/esm/loader:316:24)
    at async loadESM (node:internal/process/esm_loader:34:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.8.0

Time taken to run the queries on ./repositories/Django/thread14/wagtail_wagtail-localize/wagtail-wagtail-localize-f005601: 213.122 seconds.
Time taken to run the queries on ./repositories/Django/thread14/welliamcao_VManagePlatform/welliamcao-VManagePlatform-24c0abe: 2049.477 seconds.
Time taken to run the queries on ./repositories/Django/thread14/yomguy_TimeSide/yomguy-TimeSide-1b104ae: 2274.674 seconds.
Time taken to run the queries on ./repositories/Django/thread14/zuri-training_PROJ-31-debpay/zuri-training-PROJ-31-debpay-60805d1: 1250.377 seconds.
Time taken to run the queries on ./repositories/Django/thread14/zuri-training_Proj-T_C_Gen-Team-74/zuri-training-Proj-T_C_Gen-Team-74-4d17152: 1089.49 seconds.
Time taken to run the queries and generate the statistics: 137850.333 seconds.
